src/bin/add_filter.rs

```diff
@@ -49,17 +54,20 @@ async fn main() {
         "name": args.name,
         "active": true,
         "user_id": "cli",
-        "catalog": format!("{}_alerts", survey),
+        "survey": survey.to_string(),
         "permissions": {
             "ZTF": [1, 2, 3],
         },
         "fv": [
             {
                 "fid": "v2e0fs",
-                "pipeline": filter_pipeline
+                "pipeline": filter_pipeline,
+                "created_at": now_jd(),
             }
         ],
         "active_fid": "v2e0fs",
+        "created_at": now_jd(),
+        "updated_at": now_jd(),
     };
```

Comment:
In general I think this could all be done much more elegantly. This binary is very deprecated and existed in a world where we didn't have structs to define filters yet. An even better solution would be to check how things are now done in the API (caus that's what we now use to post filters, and therefore is up to date).

Reply:
did you ever have a chance to look at this once @antoine-le-calloch ?